subprojects/core-api/src/main/java/org/gradle/api/artifacts/DependencySubstitutions.java

```diff
@@ -92,7 +92,7 @@ public interface DependencySubstitutions {
      * DSL-friendly mechanism to construct a dependency substitution for dependencies matching the provided selector.
      * <p>
      * Examples:
-     * <pre class='autoTested'>
```

Comment:
Why was this changed?

Reply:
Apparently we never ran this auto-tested snippet. One of my changes to the way we detect them made this start running. 

When auto-tested, this fails with the error of not being able to find the `:api` project, as the snippet tries to add a substitution for that project